mobile/mobile-core/device-discovery.ts

```diff
@@ -144,7 +144,10 @@ class IOSDeviceDiscoveryStub extends DeviceDiscovery {
 	}
 
 	public startLookingForDevices(): IFuture<void> {
-		this.$logger.warn(this.error);
+		if(this.error) {
```

Comment:
but if you execute  $ appbuilder device, you do not have any idea, that we'll execute ios related code and for me, such warning will seem strange